src/main/java/net/sf/jabref/model/groups/SearchGroup.java

```diff
@@ -151,42 +159,13 @@ public boolean isDynamic() {
     }
 
     @Override
-    public String getDescription() {
-        return this.query.getDescription();
-    }
-
```

Comment:
Could you please leave these methods. I think they are used in the `Edit Groups Dialog` and if they are not used, then this is probably a bug.